tree/dataframe/src/RDataFrame.cxx

```diff
@@ -1059,12 +1059,15 @@ all_hs.GetKeys(); // returns {"nominal", "pt:down", "pt:up", "eta:0", "eta:1"}
 Note how we passed the integer `2` instead of a list of variation tags to the second Vary() invocation: this is a
 shorthand that automatically generates tags 0 to N-1 (in this case 0 and 1).
 
-\note As of v6.26, VariationsFor() and RResultMap are in the `ROOT::RDF::Experimental` namespace, to indicate that these
+\note As of v6.30, VariationsFor() and RResultMap are in the `ROOT::RDF::Experimental` namespace, to indicate that these
```

Comment:
Pinning this to a version gets outdated pretty quickly (as we are seeing already now), maybe I would change the wording to `Currently, ...` also in the similar sentence below

Reply:
makes sense!